app/main.py

```diff
@@ -1 +1,55 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True,
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            is_hungry: bool = True,
+    ) -> None:
+        super().__init__(name, appetite=3)
+        self.is_hungry = is_hungry
```

Comment:
There is no need to redefine `self.is_hungry`, pass `is_hungry` value in the `super().__init__(` here and in the `Dog` class